### PR TITLE
Omicron vaccine multipliers

### PR DESCRIPTION
**Pull request by amichaelcohn**
Added new multipliers for boosted vaccines + Omicron

### CODE REVIEW

**Comment by octern** [comment]
Note: The PR is from amichaelcohn, which is my work account -- I goofed and made the PR from the wrong browser.

**Review by beshaya — APPROVED** [review]
Thanks for taking care of this!

We also need to update the changelog: https://github.com/microCOVID/microCOVID/blob/main/src/posts/paper/99-changelog.ts

I've added a section to the Paper doc that could be used there. You can do that in this PR or a follow-on

**Comment by beshaya** [comment]
Oh, also should add an entry to the mini-changelog on the calculator page.

**Comment by octern** [comment]
Changelogs updated! Thanks for providing the text (and for doing the research in the first place!)

File: src/locales/en.json [hunk 1]
```diff
@@ -95,8 +96,8 @@
       "call_to_action": "Calculate approximate risk of contracting COVID",
       "delta_warning": "Risks have increased substantially, including for vaccinated people. <1>See blog post</1> for more details.",
       "delta_warning_heading": "Now updated for the Delta variant",
-      "omicron_warning": "Research is in progress to determine Omicron's impact on our models. The chance of infection is likely higher than the calculator currently shows, even if you are vaccinated.",
-      "omicron_warning_heading": "Not yet updated for the Omicron variant",
+      "omicron_warning": "<p>Vaccines without boosters are counted as much less effective against infection, but <em>unboosted vaccines still reduce your risk of severe or fatal illness.</em></p><p>Also not that COVID cases are reaching record highs in some areas, making activities riskier even if you are boosted.",
```
Comment (zerotrickpony) [inline src/locales/en.json:99]: typo? "Also not" should be "Also note" i think

Comment (octern) [reply src/locales/en.json]: fixed, thanks!

**Review by octern — APPROVED** [review]

**Pull request merged**